src/utility/STM32Cube_FW/shci_tl.c

```diff
@@ -21,6 +21,8 @@
 /* Includes ------------------------------------------------------------------*/
 #include "stm32_wpan_common.h"
 
+#include <Arduino.h>
```

Comment:
Why this include is now required ?

Reply:
this is included to call the` millis()`  function inside the timeout loop